main.go

```diff
@@ -167,3 +208,17 @@ func messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
 		}
 	}
 }
+
+// Function run during the daily reminder check
+func reminderRoutine(s *discordgo.Session) error {
```

Comment:
should put this into a crontab package if we add more

Reply:
I think if we add more we can do it, but tbh im really lazy i'll add it as an issue

Reply:
https://github.com/BryanSLam/discord-bot/issues/18